unix-ffi/unittest/unittest.py

```diff
@@ -490,29 +498,34 @@ def main(module="__main__", testRunner=None):
     else:
         runner = TestRunner()
 
-    if len(sys.argv) <= 1:
-        result = discover(runner)
```

Comment:
> Note the code still has the equivalent of `if len(sys.argv) == 0: discover()` because the original had that. but I'm not sure what it would mean for sys.argv to be empty?

Do you mean this original? This was intended to catch the `len==1` mostly, main case of this is `micropython -m unittest` where `sys.argv == ['unittest']`.

It's true I don't know if there's ever a case where `len(sys.argv) == 0` unless you're manipulating `sys.argv` manually in a wrapper script, in which case all bets are off really. Not sure if I had a specific reason to make this `<= 1` rather than just `== 1` other than maybe not wanting to leave an undefined case statement.

Reply:
> Do you mean this original?

Yes exactly.

But without the check it would be unconditionally indexing `sys.argv` which is also not very nice so I'll just leave it as-is.